Sources/TiltUp/Architecture/Router.swift

```diff
@@ -47,7 +47,7 @@ public extension Router {
     }
 
     // MARK: Replacing root
-    func replaceRoot(with viewController: UIViewController, popHandler: (() -> Void)? = nil) {
+    func replaceRoot(with viewController: UIViewController, animated: Bool = true, popHandler: (() -> Void)? = nil) {
```

Comment:
looks like you aren't using this in the function, 

and nit, maybe also add for `removeAll` while adding for the others

Reply:
thanks for the catch. added it to the function and added animation to removeAll as well

Reply:
👌, realizing that in `removeAll` it was `animated: false` before, so maybe that should still be the default there (or there is no reason for it to ever be true?), similar question with `navigationController.popToRootViewController(animated: false)` in this one, just confirming it should always be false?

Reply:
Changes have been addressed.